### ISSUE TITLE
Incorrect error for vuejs-accessibility/label-has-for?

### ISSUE THREAD

**Issue by AustinGil** [vue-a11y/eslint-plugin-vuejs-accessibility#133]
Getting a weird issue with form labels. ESLint is complaining:

`Form label must have an associated control.eslint(vuejs-accessibility/label-has-for)`

But the code is correct:

```html
<label for="email">Email:</label>
<input id="email" name="email" type="email" required />
```

![image](https://user-images.githubusercontent.com/4879075/115999478-ce559e00-a5a0-11eb-8fcd-f7bdb5c7a7b8.png)

Am I missing something?

**Comment by vhoyer** [comment]
https://github.com/vue-a11y/eslint-plugin-vuejs-accessibility/issues/119

check this solution, report back if you are still having trouble

**Comment by AustinGil** [comment]
Thanks @vhoyer. Your comments in that thread helped me set it up correctly. 

I also just saw the docs where it says "To fully cover 100% of assistive devices, you're encouraged to validate for both nesting and id."

I know sometimes nesting alone doesn't work, but I've never heard of ID not being enough. Can you share any more information around when using an ID alone fails or why?

**Comment by vhoyer** [comment]
I personally can't share any use case where I've encountered such a problem. This other issue should provide some interesting information if you are interested :D https://github.com/vue-a11y/eslint-plugin-vuejs-accessibility/issues/54

The author there mentions "There seems to be no accessibility advantage to using `<label>` as a wrapper if you've already given it a `for` attribute." . That said, I don't know any of this advantage neither did the author mentioned them... so... bummer, maybe you could provide more information to that discussion, or even just agree that we should change the default :shrug: (as it causes a lot of issues to appear here)

**Comment by aitelia** [comment]
This is still not working in "eslint-plugin-vuejs-accessibility": "1.1.1". Any updates on the solution to fix this false positive issue?
![Screenshot 2022-04-28 at 11 42 47](https://user-images.githubusercontent.com/96411619/165714734-8891ce8c-c622-4ecf-8510-39ba84ca4c17.png)

**Comment by vhoyer** [comment]
> This is still not working in "eslint-plugin-vuejs-accessibility": "1.1.1". Any updates on the solution to fix this false positive issue?
> ![Screenshot 2022-04-28 at 11 42 47](https://user-images.githubusercontent.com/96411619/165714734-8891ce8c-c622-4ecf-8510-39ba84ca4c17.png)
> 
> 

Have you checked the #119 yet, @aitelia?

**Comment by aitelia** [comment]
@vhoyer yes, I have found that workaround eventually.
